src/main/webapp/partials/instructorHelpSessions.jsp

```diff
@@ -6932,6 +6932,8 @@
           <br>
 
           <br> To setup the question, enter the main question text, and add the options for the students to rank. You can configure if students can give the same rank multiple times.
+                    <br> You can also configure the minimum and (or) maximum options to be ranked. When the minimum options to be ranked restriction is enabled, it will ensure that the respondent ranks at least the number of options mentioned in the restriction.
+                    <br> Similarly, when maximum options to be ranked restriction is enabled, it will ensure that the respondent ranks at most the number of options mentioned in the restriction.
```

Comment:
@sujeet14108 do be careful and review your own changes before asking for review. You should be able to tell what is not right about the above two lines.